mailchimp3/__init__.py

```diff
@@ -203,3 +204,5 @@ def __init__(self, *args, **kwargs):
         # Templates
         self.templates = Templates(self)
         self.templates.default_content = TemplateDefaultContent(self)
+        # Customer Journey
+        self.journeys = self.customer_journeys = CustomerJourney(self)
```

Comment:
Formatting: This block belongs above near 142 right above the E-Commerce comment, see the other note at the top of this file for the why.